airbyte-config/models/src/main/java/io/airbyte/config/helpers/LogConfigs.java

```diff
@@ -4,25 +4,24 @@
 
 package io.airbyte.config.helpers;
 
+import io.airbyte.config.storage.CloudStorageConfigs;
+
 /**
- * Configuration required to retrieve logs. This is a subset of the methods defined in
- * {@link io.airbyte.config.Configs} so actual look up can be delegated in {@link LogConfiguration}.
- * This prevents conflicting configuration existing at once.
+ * Describes logging configuration. For now it just contains configuration around storage medium,
+ * but in the future will have other configuration options (e.g. json logging, etc).
  */
-public interface LogConfigs {
```

Comment:
this interface just felt like extra code. happy to bring it back if someone disagrees.